pypesto/sampling/geweke_test.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+from scipy.stats import norm
+
+
+def spectrum(x: np.array,
+             nfft: int = None,
+             nw: int = None):
```

Comment:
```suggestion
             nw: int = None) -> float:
```
(i.e. add typehint what the function returns. holds for all functions)